getting-started/project-setup.adoc

```diff
@@ -130,7 +128,7 @@ image::automatic-layout.png[Automatic Layout]
 +
 image::dmn-first-view.png[DMN First view]
 
-. If you click *Age* and then click the *Properties* icon on the right, you will open a pane for the input.
+. If you click the *Age* input node and then click the *Properties* icon on the right, you will open a pane for the input.
```

Comment:
I think officially DMN standard does not have the term input node, just Input Data node.